### DISCUSSION TITLE
Multiline placeholder

### DISCUSSION THREAD

**Discussion by doneumark** [securingsincity/react-ace#736] [Q&A]
# Problem
Defining multiline placeholder is not possible with '/n' chars.
I would like to set a property of a placeholder, that has multiline code segment. but non of the chars I though will make a line break wouldn't work (<br>, /n, so on...)
How would I be able to achieve that goal?

## Sample code to reproduce your issue
`<AceEditor placeholder="<html>\n<body>\n</body>\n<html>" />`

**Comment by vig7** [comment]
using something like placeholder={"line 1\nline 2"} and setting the white-space style to pre worked for me

**Comment by umangsavaliya** [comment]
`.ace_placeholder {
    white-space: pre;
    text-indent: 0;
    margin-left: 10px;
}`
